components/governance/org.wso2.carbon.governance.taxonomy.ui/src/main/resources/web/taxonomy/source_taxonomy.jsp

```diff
@@ -0,0 +1,173 @@
+                    + "\t\t\t<node id=\"platformTG\" displayName=\"Platform TG\">\n"
+                    + "\t\t\t\t<node id=\"asCarbon\" displayName=\"AS/Carbon\"></node>\n"
+                    + "\t\t\t\t<node id=\"dS\" displayName=\"DS\"></node>\n"
+                    + "\t\t\t\t<node id=\"developerStudio\" displayName=\"Developer Studio\"></node>\n"
+                    + "\t\t\t\t<node id=\"uiUX\" displayName=\"UI/UX\"></node>\n"
+                    + "\t\t\t\t<node id=\"platformExtension\" displayName=\"Platform Extension\"></node>\n"
+                    + "\t\t\t</node>\n" + "\t\t\t<node id=\"integrationTG\" displayName=\"Integration TG\">\n"
+                    + "\t\t\t\t<node id=\"esbGwLb\" displayName=\"ESB/GW/LB\"></node>\n"
+                    + "\t\t\t\t<node id=\"mb\" displayName=\"MB\"></node>\n"
+                    + "\t\t\t\t<node id=\"bpsBrs\" displayName=\"BPS/BRS\"></node>\n"
+                    + "\t\t\t\t<node id=\"uiUX\" displayName=\"PC \"></node>\n"
+                    + "\t\t\t\t<node id=\"platformExtension\" displayName=\"DIS\"></node>\n" + "\t\t\t</node>\n"
+                    + "\t\t\t<node id=\"dataTG\" displayName=\"Data TG\">\n"
+                    + "\t\t\t\t<node id=\"dasDss\" displayName=\"DAS/DSS\"></node>\n"
+                    + "\t\t\t\t<node id=\"cep\" displayName=\"CEP\"></node>\n"
+                    + "\t\t\t\t<node id=\"ml\" displayName=\"ML\"></node>\n"
+                    + "\t\t\t\t<node id=\"analytics\" displayName=\"Analytics\"></node>\n"
+                    + "\t\t\t\t<node id=\"research\" displayName=\"Research\"></node>\n" + "\t\t\t</node>\n"
+                    + "\t\t\t<node id=\"apiTG\" displayName=\"API TG\">\n"
+                    + "\t\t\t\t<node id=\"apiManager\" displayName=\"API Manager\"></node>\n"
+                    + "\t\t\t\t<node id=\"appManager\" displayName=\"App Manager\"></node>\n"
+                    + "\t\t\t\t<node id=\"emmIot\" displayName=\"EMM/IOT\"></node>\n" + "\t\t\t</node>\n"
+                    + "\t\t\t<node id=\"qaTG\" displayName=\"QA TG\">\n"
+                    + "\t\t\t\t<node id=\"qa\" displayName=\"QA\"></node>\n"
+                    + "\t\t\t\t<node id=\"qaa\" displayName=\"QAA\"></node>\n" + "\t\t\t</node>\n"
+                    + "\t\t\t<node id=\"cloudTG\" displayName=\"Cloud TG\">\n"
+                    + "\t\t\t\t<node id=\"appFactory\" displayName=\"AppFactory /SS\"></node>\n"
+                    + "\t\t\t\t<node id=\"cloudTeam\" displayName=\"Cloud Team\"></node>\n"
+                    + "\t\t\t\t<node id=\"paas\" displayName=\"PaaS\"></node>\n"
+                    + "\t\t\t\t<node id=\"devOpsTeam\" displayName=\"DevOps Team\"></node>\n" + "\t\t\t</node>\n"
+                    + "\t\t</node>\n" + "\t\t<node id=\"Finance\" displayName=\"Finance\"></node>\n"
+                    + "\t\t<node id=\"Admin\" displayName=\"Admin\"></node>\n" + "\t</root>\n" + "</taxonomy>";
+
+        }
+
+
+
+    } catch (Exception e){
```

Comment:
Thank you Sameera for mentioning it.
This has been fixed with https://github.com/wso2/carbon-governance/pull/253